### PR TITLE
Feature/scale circuits billion users

### PR DESCRIPTION
**Pull request by daodesigner**

### CODE REVIEW

File: contracts/tests/maciFactory.ts [hunk 1]
```diff
@@ -34,18 +34,18 @@ describe('MACI factory', () => {
 
   it('sets default MACI parameters', async () => {
     const { maxUsers, maxMessages, maxVoteOptions } = await maciFactory.maxValues()
-    expect(maxUsers).to.equal(15)
-    expect(maxMessages).to.equal(15)
-    expect(maxVoteOptions).to.equal(24)
+    expect(maxUsers).to.equal(4294967295)
+    expect(maxMessages).to.equal(4294967295)
```
Comment (samajammin) [inline contracts/tests/maciFactory.ts:38]: Wow, so these new circuits enable these user numbers??

Comment (samajammin) [reply contracts/tests/maciFactory.ts:38]: 🤩

**Review by samajammin — APPROVED** [review]
Thanks @daodesigner!

Mind giving a quick summary on this? You're adding larger circuits that allow for much more messages/users/vote options?

File: contracts/tests/maciFactory.ts (hunk 1 above)
Comment (corwintines) [reply contracts/tests/maciFactory.ts:38]: Neat

**Comment by samajammin** [comment]
Hey @weijiekoh - do you mind reviewing this? I'm certainly not qualified to evaluate these changes 😅

**Review by weijiekoh — APPROVED** [review]
Looks good to me!

**Comment by samajammin** [comment]
@daodesigner wasn't sure how to fix merge conflicts in this PR - I forked off & open up #257.

Will close this out & merge that. Thanks again for adding this 🚀

**Pull request closed without merging**